app.js

```diff
@@ -1,3 +1,13 @@
 const express = require('express');
 const app = express();
-app.use(express.json);
\ No newline at end of file
+const {
+    getCategories
+} = require('./controllers');
+const {
+    handle404Errors
+} = require('./errors');
+
+app.get('/api/categories', getCategories);
+app.all('*', handle404Errors);
+
```

Comment:
In your next branch it's worth adding a 500 error handler at the bottom of your app to catch any 'unhandled' errors